### ISSUE TITLE
Channel up and Down?

### ISSUE THREAD

**Issue by truxntrax** [amosstorkey/virgintivocec#2]
Followed your instructions and got this working.  Thanks so much for taking the time to share your code!

I have one q,  Is there any way on the tv remote to map the channel up and down button to the channel up and down on the VM tivo?

T

**Comment by amosstorkey** [comment]
Apologies for the delay on this. Things have been rather hectic recently.

For my remote, this is mapped, but things can vary across different TVs/remote.

There are two stages you need to do.

1) Work out the correct codes that your remote uses.
2) Edit the code to match the remote keys.

I'll try to add how to do (1) and (2) shortly.

Amos

**Comment by truxntrax** [comment]
Thanks!

**Comment by scottcoutts78** [comment]
Hello  Amos,

Ok so new to this but have managed to get the code working and can do the basics - UP, DOWN, LEFT, RIGHT, SELECT, STOP, PLAY. All the default ones listed on MKULKE's github.

So now I need to map stuff like TIVO, GUIDE, DISPLAY etc but not having any success with my LG remote.

So I managed to do part 1 and have hacked the codes for my remote. I now have all the codes and all I need to do is map them to the function of the remote keys. (It also seems like CHANNEL UP, DOWN and the NUM keys are not valid inputs from LG's CEC)

EDIT

Thanks to Amos, he advised me to convert my binary code to HEX and then look up the commands at;

https://github.com/Pulse-Eight/libcec/blob/master/include/cectypes.h

Then all I had to do was match the command to the IRCODE being sent in the code. 

On my TV though the numbers, channel up and channel down are not sent via CEC. I was able to program other buttons though to do major functions like guide, my shows, record and power. 

Really happy with the results!

Thanks Amos, Scott
